### PR TITLE
fixed rich card rendering with product requirements

### LINKED ISSUE #911: fix rendering of rich cards with product requirements
requirements:
- The media template should be designed on the UI to have at least one image and one button
- The button link could be an external link or a trigger for a suggested reply
- Title and description are optional fields, so the UI should not break in case these fields were missing

### PR DESCRIPTION
**Pull request by AudreyKj**
resolves #911 / related to #858

note: atm buttons contain external links, perhaps this would have to be fixed in the future if we want the buttons to link to another template in addition to external links

changes: 
- Title, description, and one suggestion button are optional fields
 - The template is designed to have at least one image and one button
- time is removed from the component (as we will externalize this)

### CODE REVIEW

File: lib/typescript/render/components/RichCard/index.tsx [hunk 1]
```diff
@@ -3,40 +3,52 @@ import styles from './index.module.scss';
...
 import {DefaultMessageRenderingProps} from '../index';
 
-interface Suggestion {
-  reply: {
-    text: string;
-    postbackData: string;
-  };
-}
+type Suggestions = [
+  {
```
Comment (bitboxer) [inline lib/typescript/render/components/RichCard/index.tsx:7]: Short question: can there only be  two suggestions? Or is this possibly an array of suggestions?

**Review by bitboxer — APPROVED** [review]
LGTM, one little question.

File: lib/typescript/render/components/RichCard/index.tsx (hunk 1 above)
Comment (AudreyKj) [reply lib/typescript/render/components/RichCard/index.tsx:7]: there can only be two suggestions maximum (=2 buttons) and at least one suggestion

Comment (bitboxer) [reply lib/typescript/render/components/RichCard/index.tsx:7]: Ah, understood.

**Pull request merged**